gatsby_fp/src/styles/facultyprofile.css

```diff
@@ -88,6 +83,13 @@ hr.top-bar{
   padding: 0;
 }
 
+.title > p{
+  font-family: "Chaparral W01",Georgia,Times,"Times New Roman",serif;
```

Comment:
This doesn't seem to achieve anything, because we have the same exact property on `body` (line 3 of this file) and there's nothing else getting in the way of that.